### ISSUE TITLE
Gottron sign

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10222]
**Preferred term label:**

**Synonyms**
PMID 23112358: Violaceous patches overlying the knees

**Definition (free text, please give PubMed ID)**
PMID: 31279808 Gottron sign: Erythematous macules or patches over extensor surfaces of elbows, knuckles, knees, and ankles
PMID: 23112358 Gottron's sign refers to erythematous macules and patches overlying the elbows and/or knees, and are less specific findings for DM
PMID: 32644343: Gottron sign: erythematous macules or patches over the elbows or knees

**Parent term (use hpo.jax.org/app)**
Macule HP:0012733

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:221 Dermatomyositis

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
Added as HP:0430033
